src/playbooks/deploy/metadata.obsah.yaml

```diff
@@ -47,6 +47,18 @@ variables:
     help: User of the Pulp database.
   pulp_database_password:
     help: Password for the Pulp database.
+  foreman_logging_level:
+    help: Set the log level for Foreman.
+    choices:
+      - fatal
+      - error
+      - warn
+      - info
+      - debug
+  foreman_loggers:
+    help: Enable specific loggers for debugging Foreman.
```

Comment:
Should we list the choices the same way as we do for the `foreman_logging_level`?